### PR TITLE
Added new logs

### PR DESCRIPTION
**Pull request by hbcarlos**
Added hooks for exception, trace, table and onerror.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
This looks good to me!

File: src/index.ts [hunk 1]
```diff
@@ -150,47 +150,161 @@ const extension: JupyterFrontEndPlugin<void> = {
       }
     });
 
+    window.onerror = function(msg, url, lineNo, columnNo, error): boolean {
```
Comment (jtpio) [inline src/index.ts:153]: Maybe we want to make this function an arrow function to keep the same style for all hooks?

**Comment by jtpio** [comment]
Nice, thanks @hbcarlos!

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**